script/Arcueid/Day 6/06_00_ARC06_1_2.txt

```diff
@@ -307,13 +308,13 @@ She has a dignified posture, a clear tone of voice, and eyes that never waver. E
 [sha:2380e70d4dfa6bebe9c8f70615956ff3ef0959d4]{
 -- Page 104, Offset 14177.
 -- がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
-// がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
-Akiha leaps to her feet, banging her shin against the table as she does so.
+// Seems a little superfluous
```

Comment:
what does? unless you mean the comment. then you can delete it, you're right. No need for an extra comment though

Reply:
I messed up saving this line, but it's pretty minor so I'll just leave as is